judgments/tests/test_courts.py

```diff
@@ -33,7 +33,7 @@ def test_when_court_with_param_exists_and_no_dates_in_db_and_start_end_different
     ):
         get.side_effect = CourtDates.DoesNotExist
         court = self.mock_court_dates(2011, 2012)
-        self.assertEqual(get_court_date_range(court), "2011 &ndash; 2012")
+        self.assertEqual(get_court_date_range(court), "2011&nbsp;&mdash;&nbsp;2012")
```

Comment:
I'm a little confused by these, as the implementation above seems to return 'XXXX to YYYY' instead of 'XXXX – YYYY' - do they still pass? if so it might be worth checking the right thing is being tested!